.changeset/config.json

```diff
@@ -7,7 +7,13 @@
     }
   ],
   "commit": false,
-  "linked": [["@commercetools-backend/*", "@commercetools-frontend/*"]],
+  "linked": [
+    [
+      "@commercetools-backend/*",
+      "@commercetools-frontend/*",
+      "!@commercetools-frontend/codemod"
```

Comment:
Btw, this essentially decouples the codemod package versions from the rest.

I think for codemods we can keep its own version.